app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
+attributes :content_view_id, :content_view_name
+attributes :lifecycle_environment_id, :lifecycle_environment_name
```

Comment:
i was more following the 'host' conventions.  Foreman tends to do  association_id and association_name rather than
`association: {name: foo, id: 1}`